libs/sdk-ui-kit/src/Header/typings.ts

```diff
@@ -103,6 +106,13 @@ export interface IHeaderUpsellButtonProps {
     onUpsellButtonClick?: (e: React.MouseEvent) => void;
 }
 
+/**
+ * @internal
+ */
+export interface IHeaderInviteButtonProps {
```

Comment:
Please rename `IHeaderInviteButtonProps` → `IHeaderInviteProps`, as the item is not a button, is it?